Server/JokesOnYou.Web.Api/Controllers/AuthController.cs

```diff
@@ -7,16 +10,53 @@
 
 namespace JokesOnYou.Web.Api.Controllers
 {
-    public class AuthController
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
     {
-        public ActionResult Login([FromBody] UserLoginDTO userLoginDto)
+        private readonly IAuthService _authService;
+
+        public AuthController(IAuthService authService)
         {
-            throw new NotImplementedException();
+            _authService = authService;
         }
 
-        public Task<ActionResult> Register([FromBody] UserRegisterDTO userRegisterDto)
+        [AllowAnonymous]
+        [HttpPost("login")]
+        public async Task<IActionResult> Login(UserLoginDTO userLoginDto)
```

Comment:
specify that a user will be sent back

Reply:
How ?

Reply:
Change the iactionresult to ActionResult<User. Remove the Ok() that wraps the user. This will give us type safety and an Ok response is default when sending an item back i think

Reply:
![image](https://user-images.githubusercontent.com/39444801/113650703-f41e1000-9690-11eb-9dd9-5cb070189076.png)
that didnt work but this did:
![image](https://user-images.githubusercontent.com/39444801/113650860-3ba49c00-9691-11eb-99a3-773eccc15f39.png)